app/analysis/template.hbs

```diff
@@ -46,6 +50,27 @@
         <td>{{report.task.project.name}}</td>
         <td>{{report.task.name}}</td>
         <td>{{report.comment}}</td>
+        <td>
+          {{#if report.review}}
+            {{fa-icon 'check-square-o'}}
+          {{else}}
+            {{fa-icon 'square-o'}}
+          {{/if}}
+        </td>
+        <td>
+          {{#if report.notBillable}}
+            {{fa-icon 'check-square-o'}}
+          {{else}}
+            {{fa-icon 'square-o'}}
+          {{/if}}
+        </td>
+        <td>
+          {{#if report.verifiedBy}}
+            {{fa-icon 'check-square-o' title=(concat 'Verified by ' report.verifiedBy.username)}}
```

Comment:
I think it would be better to simply show the verifier instead of a check box. The check box was mainly for usage in reschedule to set current user but in analysis we want to know who was it.

Reply:
Done!